### ISSUE TITLE
Training parameters settings of Market-1501

### ISSUE THREAD

**Issue by xiaodongww** [HRanWang/Spatial-Attention#9]
Hi, thanks for your great work.
Here is a question about training parameters. I tried to retrain on Market-1501 with the following command:
```bash
python main.py -d market -b 48 -j 4 --epochs 100 --log logs/market/ --combine-trainval --step-size 40 --data-dir Market-1501
```
and the result is as follows 
```
Mean AP: 78.7%
CMC Scores    allshots      cuhk03  market1501
  top-1          61.2%       79.6%       93.1%
  top-5          74.7%       92.7%       97.6%
  top-10         80.1%       95.5%       98.4%
```
Then I download the model provided in Baidu Cloud, and the `Mean AP` is  `81.7`:
![image](https://user-images.githubusercontent.com/33693065/60587665-9e377c80-9dc7-11e9-9766-21fd44887219.png).

I also notice that your provided model is trained for 150 epochs while the command suggested in README file is 100 epochs. So, I trained another model with the following command
```bash
python main.py -d market -b 48 -j 4 --epochs 150 --log logs/market/ --combine-trainval --step-size 40 --data-dir Market-1501
```
and this time, the result is:
```
Mean AP: 78.6%
CMC Scores    allshots      cuhk03  market1501
  top-1          61.1%       79.4%       92.8%
  top-5          74.5%       92.7%       97.5%
  top-10         79.9%       95.2%       98.4%
```

My experiment environment is Python 3.6 and PyTorch 1.0.

Could you please share the training settings (e.g. how many training epochs and when to decay the learning rate )of the model provided in Baidu Cloud?

I would be grateful if you could help me figure out the problem.
Thank you.

**Comment by zp1018** [comment]
Mean AP: 78.2%，My experiment environment is Python 3.6 and PyTorch 0.4

**Comment by bmiftah** [comment]
Hi , It is very late to follow up  .. but here is what I see - I notice that the reported result is based on re-ranking , did you try that ? Or did you just run the model just with the above command ... I have the same env't as you and go nearly identical result like yours .. I run it for about 100 epoch ..I will try to run for more and see what I will get .. 

having said this .. does anyone have some idea on the CAM is for ablation obtained ? Is there a code in the model that does this ?
